config/crd/bases/submariner.io_submariners.yaml

```diff
@@ -71,6 +71,25 @@ spec:
                 type: boolean
               globalCIDR:
                 type: string
+              healthCheckSpec:
+                properties:
+                  enabled:
+                    type: boolean
+                  intervalSeconds:
+                    description: The interval upon which HealthChecker ping packets
+                      are sent
+                    format: int64
+                    type: integer
+                  maxPacketLossCount:
+                    description: Defines the maximum packet loss the healthchecker
+                      can tolerate before we declare the connection as down
```

Comment:
```suggestion
                    description: The maximum number of packets lost at which the health checker will mark the connection as down
```

Reply:
This is a generated file , the API update should resolve this